spring-cloud-stream-core-docs/src/main/asciidoc/spring-cloud-stream-overview.adoc

```diff
@@ -1805,6 +1805,46 @@ Once we have received the message, we can validate that the component functions
 Spring Cloud Stream provides a health indicator for binders.
 It is registered under the name of `binders` and can be enabled or disabled by setting the `management.health.binders.enabled` property.
 
+== Metrics Emitter
+Spring Cloud Stream provides a module called `spring-cloud-stream-metrics` that can be used to emit any available metric from boot's `/metrics` endpoint to a named channel.
```

Comment:
Also, phrasing here - instead of 'boot's `/metrics` endpoint' we can say something more elaborate - maybe a reference to relevant documentation in Spring Boot ?